Major Update (#268)
NilPointer-Software/bdfd-wiki: src/bdscript/removeButtons.md

```
$removeButtons[message id]
```

#### Parameters 

> - `message id` `(Type : Snowflake || Flag : Required)` : It removes all buttons from a message using it's message id.

Reviewer: ```suggestion
- `message id` `(Type: Snowflake || Flag: Required)`: The message from which all buttons will be removed.
```
